Packages/vcs/Lib/editors/label.py

```diff
@@ -44,6 +46,12 @@ def __init__(self, interactor, label, dp, configurator):
 
         self.actor = get_actor(self.label, self.display)
 
+        tprop = self.actor.GetTextProperty()
+        self.real_bg = tprop.GetBackgroundColor()
+        self.real_bg_opacity = tprop.GetBackgroundOpacity()
+
+        tprop.SetBackgroundColor(contrasting_color(*tprop.GetColor()))
```

Comment:
Stores the existing background color/opacity (which aren't really in use at all, but seemed polite at the time), and swaps out with a contrasting color so we can see which labels are being edited.